frontend/src/metabase/dashboard/components/DashCard/DashCardMenu/DashCardMenu.tsx

```diff
@@ -165,12 +177,11 @@ DashCardMenu.shouldRender = ({
     question.datasetQuery(),
   );
 
-  if (isEmbed) {
-    return isEmbed;
+  if (isPublicOrEmbedded) {
+    return isPublicOrEmbedded;
```

Comment:
Then here it's saner to return `true`. If it's false, this block would be skipped anyway.

```suggestion
    return true;
```